### ISSUE TITLE
What is "umbreld" and where is its source?

### ISSUE THREAD

**Issue by kroese** [getumbrel/umbrel#1654]
In the last release (0.5.4) a new component called `umbreld` was introduced. Its distributed as a binary and downloaded during the installation. I cannot find any information in the changelogs about the purpose of this component, neither can I find its sourcecode anywhere.

Is this some sneaky attempt to move away from being an open-source project to a proprietary solution? Because it seems to be the largest change (aside from cosmetic changes) and yet its not mentioned anywhere in the announcement?

**Comment by lukechilds** [comment]
The `umbreld` source code is in the `server` subdirectory: https://github.com/getumbrel/umbrel/tree/master/server

I know it's a bit hard to follow, we're currently undergoing a major refactor and will be shipping the changes incrementally, so there are kind of two code bases on top of each other at the moment. Over time, all behaviour will move into umbreld and it'll be much easier to follow.

Thanks for the eagle eyes and keeping us honest! lol

**Comment by IMPranshu** [comment]
True that. Was trying to wrap my head around the feature implementation but hard to find a roadmap for the development.

**Comment by warioishere** [comment]
you should accept that umbrel is going closed source locking out the community. Soon you will find some abos und bullshit to buy. Get lost umbrel, its really sad what you guys with this project. 

to all people, move to citadel or to raspiblitz
